### PR TITLE
Adds missing duffelbags for some jobs and update their respective vendors and closets.

### PR DESCRIPTION
**Pull request by Tristrian**
## About The Pull Request

Some current jobs have their own backpacks, satchels and even duffelbags. Others have their own backpacks and satchels but no duffelbag. This PR adds the missing duffelbags for the following jobs : 

- Scientist / Roboticist / Research Director 
- Geneticist
- Chemist
- Virologist
- Botany
- Miner

Their vendors, closets, and roundstart preference have been updated to have these new bags.

EDIT : 
I forgot to include a screenshot at how they look : 

Duffelbags
## Why It's Good For The Game

- Fixes consistency between some jobs.
- Fashion preference
- Ability to get a (departemental) duffelbag without the need to unlock every closet or to wait for botany to make cloth when you have a project that needs to carry lots of items.

## Changelog
:cl:
add: Added duffelbags for Science, Botany, Virology, Genetics, Mining and Chemistry.
tweak: Vendors have been updated to sell the duffelbags.
tweak: The new duffelbags can be selected in the character preferences for the affected jobs.
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
add them to cargo vendor

**Comment by Fikou** [comment]
make the mining prescription kit use the mining duffelbag

**Comment by LemonInTheDark** [comment]
What a spectacular first pr

**Comment by GoldenAlpharex** [comment]
Now all we need are departemental  fanny packs :^)

**Comment by Tristrian** [comment]
> Now all we need are departemental fanny packs :^)

Maybe :]

**Comment by Jared-Fogle** [comment]
hey cool i remember asking for this

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -424,6 +424,44 @@
 	name = "surgical duffel bag"
 	desc = "A large duffel bag for holding extra medical supplies - this one seems to be designed for holding surgical tools."
 
+/obj/item/storage/backpack/duffelbag/explorer
+	name = "explorator's duffel bag"
+	desc = "A large duffel bag for holding extra exotic treasures."
+	icon_state = "duffel-explorer"
+	inhand_icon_state = "duffel-explorer"
+
+/obj/item/storage/backpack/duffelbag/hyd
```
Comment (Jared-Fogle) [inline code/game/objects/items/storage/backpack.dm:433]: We don't gain anything from short names like this. I know the other ones have similar, but please extend these out (`hydroponics`, `chemist`, `genetics`) etc.

Comment (Tristrian) [reply code/game/objects/items/storage/backpack.dm]: Changes done

**Review by Mothblocks — APPROVED** [review]
Code is good, needs art lead for merge.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-5). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**